components/raftstore-v2/src/operation/query/mod.rs

```diff
@@ -83,12 +85,23 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     /// Query internal states for debugging purpose.
     pub fn on_query_debug_info(&self, ch: DebugInfoChannel) {
         let entry_storage = self.storage().entry_storage();
-        let meta = RegionMeta::new(
+        let mut meta = RegionMeta::new(
             self.storage().region_state(),
             entry_storage.apply_state(),
             GroupState::Ordered,
             self.raft_group().status(),
         );
+        // V2 doesn't persist commit term, so fill it on demand.
```

Comment:
I don't understand this comment.

Reply:
In v2, `commit_index` and `commit_term` are always 0. So here needs to initialized the value for debug purpuse.

Reply:
```suggestion
        // V2 doesn't persist commit index and term, fill them with in-memory values.
```